tools/underlay.repos

```diff
@@ -11,6 +11,10 @@ repositories:
   #   type: git
   #   url: https://github.com/ros-simulation/gazebo_ros_pkgs.git
   #   version: ros2
+  # ros-perception/image_common:
```

Comment:
This was present in the original PR (#2111) but my guess is that its no longer needed

Reply:
Yeah, remove it all